Wikimate.php

```diff
@@ -1419,10 +1420,11 @@ public function newSection($name, $text)
     /**
      * Deletes the page.
      *
-     * @param   string   $reason  Reason for the deletion
-     * @return  boolean           True if page was deleted successfully
+     * @param   string   $reason   Reason for the deletion
+     * @param   boolean  $talktoo  True to delete talk page too (MW 1.38+)
+     * @return  boolean            True if page was deleted successfully
      */
-    public function delete($reason = null)
+    public function delete($reason = null, $talktoo = false)
```

Comment:
I find it odd that [the default in MediaWiki is false](https://github.com/wikimedia/mediawiki/blob/1.38.0/includes/api/ApiDelete.php#L175). Would you be open to deviate from that behavior and default to deleting the talk page too? I can't think of many cases where one would want to delete the main page but keep its talk page around, but maybe I just lack imagination! :sweat_smile:

Reply:
Historically talk pages were not deleted while deleting a main/file space page, this required a separate talk delete, so the default makes sense. And a common situation (at least on the Doom Wiki) is that the talk page has a discussion/vote about the article deletion, which is then preserved to document the voting result. So I definitely don't think defaulting to `true` is a good angle for Wikimate.

Reply:
Ah, that does make sense, yeah. Ok then, let's follow suit then.